### PR TITLE
Added passphrase input for SSH

### PR DESCRIPTION
**Pull request by akominch**
This PR closes issue https://github.com/jesseduffield/lazygit/issues/781

### CODE REVIEW

File: pkg/i18n/dutch.go [hunk 1]
```diff
@@ -55,6 +55,9 @@ func addDutch(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "CredentialsPassword",
 			Other: "Wachtwoord",
+		}, &i18n.Message{
+			ID:    "CredentialsPassphrase",
+			Other: "Voer wachtwoordzin in voor SSH-sleutel",
```
Comment (mjarkk) [inline pkg/i18n/dutch.go:60]:
```suggestion
			Other: "Voer wachtwoord in voor SSH-sleutel",
```
I think this is a better sentence

**Comment by jesseduffield** [comment]
I'm not sure how to test this. Is anybody able to double check that this works? @mjarkk @glvr182 @dawidd6

**Comment by glvr182** [comment]
Does not seem to work at my end.
Looking at the files this PR only pops up a prompt for the password (which it didnt in my case), but I don't understand where the password gets send to the ssh-agent.

2020-04-27-112439_941x1028_scrot

**Comment by mjarkk** [comment]
@jesseduffield  
I think you can test it the following way:  
- Create a new ssh key via `ssh-keygen` and make sure to add a passphrase
- Add the key as ssh key to a git server for example github, gitlab, gitea, whatever.  (make sure this is your only ssh key in the git server)
- clone a ssh repo from the git server and try some things in lazygit it should hang or ask for credentials

**Comment by jesseduffield** [comment]
I can actually test this now cos I've got my own remote ssh git server at home! I'll hopefully test this today

**Comment by jesseduffield** [comment]
I gave this a test and it didn't work. My prompt was more like 'blah@192.168.1.12's password:'. I'm not sure how to get the right prompt to test this

**Comment by mjarkk** [comment]
I've put up a MR that theoretically should solve all these errors: https://github.com/jesseduffield/lazygit/pull/984

**Pull request closed without merging**